contents.json

```diff
@@ -3236,6 +3240,11 @@
       "category": "ui",
       "description": "Easy way to make elements in your iOS app twinkle.",
       "homepage": "https://github.com/piemonte/Twinkle"
+    }, {
+        "title": "Insert3D",
+        "category": "ui-3d",
+        "description": "The fastest 🚀 way to embed a 3D model in Swift.",
```

Comment:
Hi @Viktoo, could you remove "in Swift", according to this rule: "Description does not say "written in Swift" or variant 🤓"?

Reply:
Absolutely! On it!